gui/jails/forms.py

```diff
@@ -468,6 +475,21 @@ def save(self):
         self.instance = Jails.objects.get(jail_host=jail_host)
 
 
+def is_JailRoot_Shared(jail_root):
```

Comment:
Not trying to be annoying here, but that name seems like a javascript camel case! :D 
Since you might have other things to change, can you consider using snake case instead of camel case? e.g. is_jailroot_shared or is_jail_root_shared?
This is just my OCD speaking, lol :)